src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotProfilingInfoImpl.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 2012, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
this is one is fine since you copied it from an other file.

Reply:
Fixed.